airflow/www_rbac/views.py

```diff
@@ -1245,23 +1249,27 @@ def set_duration(tid):
     def graph(self, session=None):
         dag_id = request.args.get('dag_id')
         blur = conf.getboolean('webserver', 'demo_mode')
-        dag = dagbag.get_dag(dag_id)
-        if dag_id not in dagbag.dags:
-            flash('DAG "{0}" seems to be missing.'.format(dag_id), "error")
-            return redirect('/')
-
         root = request.args.get('root')
-        if root:
-            dag = dag.sub_dag(
-                task_regex=root,
-                include_upstream=True,
-                include_downstream=False)
+        if root: show_dag_id = root
```

Comment:
Since the entire Airflow codebase places the statements in an if/else on a newline, I'd stick to that for consistency.

```
if root:
    show_dag_id = root
else:
    show_dag_id = dag_id
```

Reply:
fixed now